x11/xcowsay/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        nickg xcowsay 1.4 v
+revision            1
+github.tarball_from releases
+name                xcowsay
+categories          x11
+license             GPL-3+
+platforms           darwin
+maintainers         nomaintainer
+description         X11 xcowsay app
```

Comment:
This is not a good description. You could use the first sentence of the long_description.